src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -310,8 +313,11 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, correlat
                                          "is already onboarded to the resource group" +
                                          " '{}' with resource name '{}'.".format(configmap_rg_name, configmap_cluster_name))
         else:
-            # Cleanup agents and continue with put
-            utils.delete_arc_agents(release_namespace, kube_config, kube_context, helm_client_location, is_arm64_cluster)
+            logger.warning("Cleaning up the stale onboarding resource present in the cluster to continue with fresh onboarding.")
```

Comment:
nit - "Cleaning up the stale arc agents present on the cluster so that a fresh onboarding can be started."